cmd/auth/auth.go

```diff
@@ -34,25 +35,23 @@ GCP: https://docs.gcp.databricks.com/dev-tools/auth/index.html`,
 }
 
 func promptForHost(ctx context.Context) (string, error) {
+	if !cmdio.IsInTTY(ctx) {
+		return "", fmt.Errorf("the command is being run in a non-interactive environment, please specify a host using --host")
+	}
+
 	prompt := cmdio.Prompt(ctx)
 	prompt.Label = "Databricks Host (e.g. https://<databricks-instance>.cloud.databricks.com)"
```

Comment:
This PR low-cases words that don't need capitalization, so it should rename `Host` -> `host` as well.

Reply:
Done